bin/util.js

```diff
@@ -212,7 +171,7 @@ exports.sign = async function(program) {
   return message;
 };
 
-exports.verify = async function(program) {
+exports.verify = async function(program, requestOptions, publicKeyFile) {
   /**
  * 1. recreate the canonzied string (not hashed, not signed, not base 64)
```

Comment:
this isn't new, but looks like indent on this comment is off if it's supposed to be following jsdoc convention.

Reply:
Addressed in https://github.com/digitalbazaar/http-signature-header/pull/12/commits/c3db9340863113950472faa881437a6dcc154eaa